conf/services/tax-credits-renewals.yml

```diff
@@ -1,11 +1,51 @@
 serviceName: Tax Credits Renewals
 serviceDescription: |
-  The Tax Credits Renewals service can be used by anyone currently receiving tax credits to renew their claim during the
-  April to August renewals window.
+  The Tax Credits Renewals service is for anyone receiving tax credits to renew their claim during the April to August renewals window.
 serviceDomain: www.tax.service.gov.uk
 serviceUrl: /tax-credits-service/renewals
 contactFrontendServiceId: tax-credits-renewals
 complianceStatus: noncompliant
-statementCreatedDate: 2021-03-22
 statementVisibility: public
-statementLastUpdatedDate: 2021-03-22
\ No newline at end of file
+statementCreatedDate: 2021-03-22
+serviceLastTestedDate: 2021-06-30
```

Comment:
this date is different from the english date... is this correct?

Reply:
Nope. I had to reformat the dates from the Word doc and I clearly messed those up. Ta.